owncloud/owncloud.py

```diff
@@ -589,7 +590,7 @@ def __put_file_chunked(self, remote_path, local_source_file, **kwargs):
                 headers=headers
             )
 
-        chunk_count = size / chunk_size
+        chunk_count = math.ceil(size / chunk_size)
```

Comment:
This will behave different in python 2 and python 3.
Python 2 - math.ceil(10/3) == 3.0
Python 3 - math.ceil(10/3) == 4.0

Doing
 ```using from __future__ import division ```
The division in python2 results float. Not sure if adding this will break other things.